pages/common/firefox.md

```diff
@@ -0,0 +1,24 @@
+# firefox
+
+> A free and open source web browser.
+> More information at https://developer.mozilla.org/en-US/docs/Mozilla/Command_Line_Options.
+
+- Open firefox at www.duckduckgo.com:
+
+`firefox http://www.duckduckgo.com`
```

Comment:
http://www.duckduckgo.com should be inside tokens (`{{}}`)

Reply:
Can we also use HTTPS? 🤔